Stripe/STPEmailAddressValidator.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  STPEmailAddressValidator.swift
+//  Stripe
+//
+//  Created by Jack Flintermann on 3/23/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+import Foundation
+
+class STPEmailAddressValidator: NSObject {
+  class func stringIsValidPartialEmailAddress(_ string: String?) -> Bool {
+    guard let string = string else {
+      return false
```

Comment:
I think this should be true (nil is 0 which is <=1)

Reply:
++, that isn't how it worked before, but I confirmed that it won't break anything to fix